tutorials/rnn/ptb/ptb_word_lm.py

```diff
@@ -177,7 +177,8 @@ def _build_rnn_graph(self, inputs, config, is_training):
   def _build_rnn_graph_cudnn(self, inputs, config, is_training):
     """Build the inference graph using CUDNN cell."""
     inputs = tf.transpose(inputs, [1, 0, 2])
-    self._cell = tf.contrib.cudnn_rnn.CudnnLSTM(
+    from tensorflow.contrib.cudnn_rnn.python.ops import cudnn_rnn_ops
```

Comment:
Why?

Reply:
I don't know why there are two versions of `CudnnLSTM` - `cudnn_rnn.python.ops.cudnn_rnn_ops.CudnnLSTM` and `cudnn_rnn.python.layers.cudnn_rnn.CudnnLSTM`, and the API is different. It looks like this tutorial was written using the version in `cudnn_rnn_ops`, since it is using the `input_size` parameter.